### PR TITLE
attempt to toggle pause button

### PR DESCRIPTION
**Pull request by hsoc**
Haven't tested it. It "should" work (c) :-p

### CODE REVIEW

**Comment by sirodoht** [comment]
Hey @hsoc! Thanks for the PR!

So, there are two issues with the code. The first is that in lines 184 and 186 you don't actually call those functions. You would need to use parenthesis to actually call them, i.e. `this.start()` instead of `this.start`.

The second problem is that in line 183 you use the function `this.start` to check if the timer is stopped or not. This won't work because the start function does not return something that can be evaluated as either true or false. (also the same problem applies as before, you need to use `()`)

So, the if statement in 183 will always evaluate as false, because `this.start` will always return the function object, which is true, and with `!` it will become false

**Comment by hsoc** [comment]
Oops. :-) @sirodoht

File: app/main.js [hunk 1]
```diff
@@ -163,9 +163,10 @@ const Timer = React.createClass({
       timeRemainingMins: newTimeRemainingMins,
       timeRemainingSecs: newTimeRemainingSecs,
     })
-    this.listenForNorify()
+    this.listenForNotify()
   },
   start: function() {
+    currentTask.paused === 'false'
```
Comment (sirodoht) [inline app/main.js]: This is a variable assignment, so you need to use `=`, not the conditional operator `===`.

Also, `false` is not a string. So, just `false`, with no quotes.

File: app/main.js [hunk 2]
```diff
@@ -176,21 +177,22 @@ const Timer = React.createClass({
     }
   },
   pause: function() {
+    currentTask.paused = true
```
Comment (sirodoht) [inline app/main.js]: This is correct, but you need to find the `currentTask` first. So, just above this line, copy the line from line 172, i.e. `const currentTask = find(this.state.tasksList, {current: true})`

Then, when we assign `currentTask.paused = true`, the `currentTask` will be known.

File: app/main.js [hunk 3]
```diff
@@ -176,21 +177,22 @@ const Timer = React.createClass({
...
     clearInterval(this.intervalSec)
     clearInterval(this.intervalMin)
   },
   togl: function() {
-    if(!this.start) {
-        this.start
+    if(currentTask.paused === 'true') {
```
Comment (sirodoht) [inline app/main.js]: Same thing with the above comment [1] here. You need to find the `currentTask` first.

Also, some thing with true. Just `true`, not `'true'` (no quotes)

[1] https://github.com/sirodoht/ketchup/pull/5/files/4643cd197cc477d785a032da8b8c0d3c96101715..6441c0dbe1e239278ea4194a25d320719f3f3978#r105179502

**Review by sirodoht — CHANGES_REQUESTED** [review]
Mainly two corrections and it's done!

**Review by sirodoht — APPROVED** [review]

**Pull request merged**